data/hai ships.txt

```diff
@@ -87,7 +87,7 @@ ship "Flea"
 	engine 0 27
 	gun 0 -27 "Railgun"
 	explode "tiny explosion" 15
-	description `This is a combat drone designed to help contain Unfettered aggression with a minimum risk to Hai life.`
+	description `The Flea is a light combat ship designed to help contain Unfettered aggression, while keeping a minimum risk to Hai life using remote control rather than pilots.`
```

Comment:
`This is a combat drone designed to help contain Unfettered aggression a minimum risk to Hai life.`?